examples/parcel/.compiledcssrc

```diff
@@ -7,8 +7,16 @@
     ".tsx",
     ".customjsx"
   ],
-  "babelPlugins": [
+  "parserBabelPlugins": [
```

Comment:
Can we update the [docs](https://compiledcssinjs.com/docs/pkg-parcel-transformer) with these extra configurable options?